WordPress/src/test/java/org/wordpress/android/ui/bloggingprompts/editor/EditorBloggingPromptsViewModelTest.kt

```diff
@@ -77,4 +86,18 @@ class EditorBloggingPromptsViewModelTest : BaseUnitTest() {
         viewModel.start(siteModel, -1)
         verify(bloggingPromptsStore, times(0)).getPromptById(any(), any())
     }
+
+    @Test
+    fun `should NOT CALL block mapper if enhancements feature flag is DISABLED`() {
+        whenever(bloggingPromptsEnhancementsFeatureConfig.isEnabled()).thenReturn(false)
+        viewModel.start(siteModel, 123)
+        verify(bloggingPromptsEditorBlockMapper, times(0)).map(any())
+    }
+
+    @Test
+    fun `should CALL block mapper if enhancements feature flag is ENABLED`() {
+        whenever(bloggingPromptsEnhancementsFeatureConfig.isEnabled()).thenReturn(true)
+        viewModel.start(siteModel, 123)
+        verify(bloggingPromptsEditorBlockMapper, times(1)).map(any())
+    }
```

Comment:
**Suggestion:** I wonder if the test could validate **state** instead of **behavior** in these tests. This could be done my returning a fixed String in the `bloggingPromptsEditorBlockMapper` mock in line 53 (e.g.: `"mapped"`), then doing asserts on the `loadedPrompt.content`.

I _think_ the first test would be (disabled FF):
```kotlin
assertThat(loadedPrompt?.content).isEqualTo(bloggingPrompt.model?.content)
```

Then the second test (enabled FF) would be:
```kotlin
assertThat(loadedPrompt?.content).isEqualTo("mapped")
```

Wdyt?

---

**Request:** Also, since this loaded prompt `content` is now derived from the feature flag value, I think it might make sense to remove the assertion on line 77. Does that make sense?